load-test-framework/src/main/resources/command.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
```

Comment:
It came up somewhere in this directory, I think, where the Command.java file (the really long, do not edit file produced by the protobuf compiler). That java file is automatically produced with the maven build right? Could we remove it from the directory and add to .gitignore just to clean up the repo a little bit?